src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -248,7 +248,7 @@ DEFAULT_FROM_EMAIL: {{ key "edxapp/sender-email-address" }} # MODIFIED
 DEFAULT_MOBILE_AVAILABLE: false
 DEFAULT_SITE_THEME: mitxonline  # MODIFIED
 DEPRECATED_ADVANCED_COMPONENT_TYPES: []
-DISABLE_FORUM_V2: true
+DISCUSSIONS_ENABLE_FORUM_V2: true
```

Comment:
I'm not sure that this is actually a setting used by anything. We want to set [`ENABLE_DISCUSSION_SERVICE`](https://github.com/openedx/forum/blob/master/forum/settings/common.py#L37C9-L37C34)

Reply:
Yup we're setting that further up in the file. Removed. Thanks!